tests/icon_score/test_icon_container_db.py

```diff
@@ -339,3 +339,33 @@ def test_container_util(self):
         with self.assertRaises(InvalidParamsException):
             prefix: bytes = ContainerUtil.create_db_prefix(VarDB, 'vardb')
 
+
+class TestOnlyGetItemObj:
+    def __init__(self, limit):
+        self._limit = limit
+
+    def __getitem__(self, key):
+        if key > self._limit:
+            raise IndexError(key)
+
+
+class TestIterObj(TestOnlyGetItemObj):
+    def __iter__(self):
+        pass
+
+
+class TestContainerDB(unittest.TestCase):
```

Comment:
What is the purpose of tests? Please let me know.

Reply:
This test created a situation where iterable with getitem and it could take an infinite loop.